google/cloud/bigquery/v2/minimal/internal/common_v2_resources.cc

```diff
@@ -194,22 +195,36 @@ void from_json(nlohmann::json const& j, Value& v) {
       case 0:
         // Do not set any value
         break;
-      case 1:
-        v.value_kind = j.at("valueKind").get<double>();
+      case 1: {
+        double val;
+        SafeGetTo(val, j, "valueKind");
+        v.value_kind = val;
```

Comment:
you might want a `SafeValueOr` here and below.

https://github.com/googleapis/google-cloud-cpp/pull/12209#discussion_r1273010101